Source/ACE/Entity/Landblock.cs

```diff
@@ -729,6 +733,13 @@ public void ScheduleItemTransfer(ActionChain chain, ObjectGuid wo, ObjectGuid co
             }
         }
 
+        public bool ItemExistsInWorld(ObjectGuid wo)
```

Comment:
I just found an example - it is a different message with the same response from the server.   The client sends us a F7B1 00CD   that will make it all work.